apps/job-info-consumer/consumer/pkg/postgres.go

```diff
@@ -0,0 +1,121 @@
+package pkg
+
+import (
+	"context"
+	"database/sql"
+	"embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/bacalhau-project/bacalhau/pkg/requester/pubsub/jobinfo"
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+	"github.com/rs/zerolog/log"
+
+	_ "github.com/lib/pq"
+)
+
+type PostgresDatastoreParams struct {
+	Host        string
+	Port        int
+	Database    string
```

Comment:
Should we have an explicit sslmode parameter too? We use `sslmode=disable` below but I'm sure we'd need this on AWS.